providers/amazon/src/airflow/providers/amazon/aws/transfers/sql_to_s3.py

```diff
@@ -128,14 +134,45 @@ def __init__(
         self.aws_conn_id = aws_conn_id
         self.verify = verify
         self.replace = replace
-        self.pd_kwargs = pd_kwargs or {}
         self.parameters = parameters
-        self.read_pd_kwargs = read_pd_kwargs or {}
         self.max_rows_per_file = max_rows_per_file
         self.groupby_kwargs = groupby_kwargs or {}
         self.sql_hook_params = sql_hook_params
+        self.df_type = df_type
+        if read_pd_kwargs is not None and read_kwargs is not None:
+            raise AirflowException(
+                "Cannot specify both 'read_kwargs' and 'read_pd_kwargs'. Use 'read_kwargs' instead."
+            )
```

Comment:
I would not raise an error but use `read_kwargs` instead. And the next if ensures we are raised a deprecation warning so that the user is aware